src/citrine/_utils/functions.py

```diff
@@ -86,10 +85,10 @@ def object_to_link_by_uid(json: dict) -> dict:
         uids = json['uids']
         if not isinstance(uids, dict) or not uids:
             return json
-        if 'id' in uids:
-            scope = 'id'
+        if CITRINE_SCOPE in uids:
+            scope = CITRINE_SCOPE
         else:
-            scope = list(uids.keys())[0]
+            scope = next(iter(uids))
```

Comment:
This was a clean-up while I was looking at it.  Adding a None return with a more informative error condition if there's no scope could make sense.